src/core/core.scale.js

```diff
@@ -326,9 +341,12 @@ var Scale = Element.extend({
 		me.beforeFit();
 		me.fit();
 		me.afterFit();
-		//
+		// Auto-skip
+		me._ticks = optionTicks.display && optionTicks.autoSkip ? me._autoSkip(me.getTicks()) : me.getTicks();
```

Comment:
Here, `tickOpts.display` seems easier to understand than `optionTicks.display`. Probably just me.

Reply:
Agreed